modules/configuring-shared-cpus-for-a-workload.adoc

```diff
@@ -0,0 +1,106 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-shared-cpus-for-a-workload_{context}"]
+= Configuring shared CPUs in a performance profile
+
+You can define and enable shared CPUs by using a performance profile. You can use these shared CPUs for lightweight application tasks.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+* You enabled the `TechPreviewNoUpgrade` feature set on the cluster. For more information about enabling a feature set, see the _Additional resources_ section.
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set cannot be undone and prevents minor version updates. These feature sets are not recommended on production clusters.
```

Comment:
In case it's relevant the feature gates feature name itself is `MixedCPUsAllocation`

Reply:
I added this to the list of feature gates in the "Understanding feature gates" section.